srcpkgs/pony-stable/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'pony-stable'
+pkgname=pony-stable
+version=0.1.4
+revision=1
+only_for_archs="x86_64"
+hostmakedepends="ponyc"
+makedepends="libatomic-devel"
+short_desc="A simple dependency manager for the Pony language"
+maintainer="Brian Mitchell <brian@strmpnk.co>"
+license="2-clause-BSD"
+homepage="https://github.com/ponylang/pony-stable"
+distfiles="https://github.com/ponylang/pony-stable/archive/${version}.tar.gz"
+checksum=c9888f6d10a8d512335ddfe77e2b36bceb23226cdd694e24c57ab904bc4155fd
+
+do_build() {
+	make
```

Comment:
Why not `make ${makejobs}`?

Reply:
Good question. The makefile is a wrapper that calls ponyc which currently will not parallelize. The -j here would not do much for us as it stands.

Reply:
Ok.